packages/docusaurus-theme-classic/src/theme/Heading/styles.module.css

```diff
@@ -1,3 +1,10 @@
+/**
```

Comment:
This is the only one we should be keeping.

Reply:
What about bootstrap template files?

Reply:
They shouldn't have it too. It's just like the classic template, it shouldn't have the Facebook copyright headers because they will be *forked* by non-FB users.